Monika After Story/game/zz_backgrounds.rpy

```diff
@@ -2991,7 +2991,8 @@ init -2 python in mas_background:
             new_flt - incoming filter.
             curr_time - current time as datetime.time
         """
-        if new_flt == mspr.FLT_DAY or new_flt == mspr.FLT_NIGHT:
+        # As of now we only have these, but just in case of future changes
+        if new_flt in (mspr.FLT_DAY, mspr.FLT_NIGHT, mspr.FLT_SUNSET):
```

Comment:
maybe functionize this check? `mas_isIslandFltEnabled` (or a better name)

Reply:
I was thinking of a function like this.